### PR TITLE
Checkbox add missing return types and separate HTML and JS code in examples

### PR DESCRIPTION
**Pull request by lslachciak**
[Problem] There were no return types in method signatures
[Solution] Add missing return types

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/605

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm [hunk 1]
```diff
@@ -189,11 +191,13 @@ <h3>Summary</h3>
 							prettyprint">
 &lt;input type=&quot;checkbox&quot; name=&quot;checkbox-1&quot; id=&quot;checkbox-1&quot;/&gt;
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm]: According to [this comment](https://github.com/Samsung/tizen-docs/pull/607#discussion_r263327829),
if there are two blocks of code after each other then both descriptions HTML and JS are to be used.

How about these sample code. Don't we need "\<p>HTML code:\</p>" here?

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm [hunk 2]
```diff
@@ -132,12 +133,13 @@ <h3>Summary</h3>
 							<pre name="code" class="examplecode
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm]: According to [this comment](https://github.com/Samsung/tizen-docs/pull/607#discussion_r263327829),
if there are two blocks of code after each other then both descriptions HTML and JS are to be used.

How about these sample code. Don't we need "\<p>HTML code:\</p>" here?

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm [hunk 3]
```diff
@@ -244,15 +248,16 @@ <h3>Summary</h3>
 							prettyprint">
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm]: According to [this comment](https://github.com/Samsung/tizen-docs/pull/607#discussion_r263327829),
if there are two blocks of code after each other then both descriptions HTML and JS are to be used.

How about these sample code. Don't we need "\<p>HTML code:\</p>" here?

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm (hunk 1 above)
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm]: Fixed

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm (hunk 2 above)
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm]: Fixed

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm (hunk 3 above)
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm]: Fixed

**Pull request merged**